radish/extensions/console_writer.py

```diff
@@ -225,7 +228,16 @@ def console_writer_after_each_step(self, step):
 
         color_func = self.get_color_func(step.state)
         line_jump_seq = self.get_line_jump_seq() * (((len(step.raw_text) + 3) if step.text else 1) + (len(step.table) if step.table else 0))
-        output = u"{0}        {1}{2}".format(line_jump_seq, self.get_id_sentence_prefix(step, colorful.bold_cyan), color_func(step.sentence))
+        output = u'{0}        '.format(line_jump_seq)
+
+        if isinstance(step.parent, ScenarioOutline):
+            # Highlight Scenario Outline placeholders e.g. '<method>'
+            output += (u''.join([str(colorful.bold_white(item) if (self._placeholder_regex.search(item)
+                                 and item.strip('<>') in step.parent.examples_header)
```

Comment:
re.sub does not work unless you add the default color codes after the substitutions to change the color back to the standard color. Which is essentially what this solution accomplishes  e.g. `When I <method> the <fruit>.` . "When I", "the" and the "." need to be the standard color.